### PR TITLE
k8s deployment modifications, Build-time LLM env var

### PR DESCRIPTION
**Pull request by justinthelaw**
* Confirmed K3D + DUBBD + LFAI (Go API) + Whisper + CTransformers integration working
* Groundwork for CTransformers + Batched Summary
* Dynamic ORIGIN value based on full domain, defaulting to Istio Tenant Gateway's external IP address
* Summarization LLM switching at deploy time using env vars

### CODE REVIEW

**Comment by CollectiveUnicorn** [comment]
Does the docker deployment work properly with purely using the `.env` file?

**Comment by justinthelaw** [comment]
> Does the docker deployment work properly with purely using the `.env` file?

Yeah, the dotenv is mostly being used for pulling in the environment variables being pulled in from the Zarf build and deploy. In `openai.ts` uses one of those variable to set the leapfrogai base url, and I decided to also use that to tell doug-translate which model to use for summarization. We could just add to the docker command to use the .env file using the env flag `--env-file .env` during Docker run?

File: manifests/namespace.yaml
Comment (Noxsios) [inline manifests/namespace.yaml:1]:
```yaml
          - cmd: ./zarf tools kubectl label namespace doug-translate istio-injection=enabled
```

can safely be removed from the `zarf.yaml` with this ns addition

**Review by Noxsios — CHANGES_REQUESTED** [review]

**Review by CollectiveUnicorn — APPROVED** [review]

File: manifests/namespace.yaml
Comment (justinthelaw) [reply manifests/namespace.yaml:1]: Yeah, I saw that in the zarf.yaml, but it wasn't quite working for me at first. Let me test again to make sure! And if it does, I'll modify this PR.

File: zarf.yaml
[the remark this answers is not in the document]
Comment (CollectiveUnicorn) [reply zarf.yaml:1]: The namespace way I think is objectively better, I think the suggestion was to remove the line `./zarf tools kubectl label namespace doug-translate istio-injection=enabled` from the zarf.yaml to use what you added instead.

[the remark this answers is not in the document]
Comment (justinthelaw) [reply zarf.yaml:1]: Ah okay, I'll revert haha

**Review by CollectiveUnicorn — APPROVED** [review]

**Review by Noxsios — APPROVED** [review]

**Pull request merged**